buku

```diff
@@ -950,29 +949,34 @@ class BukuDb:
         except IndexError:
             logger.error('Index out of bound')
 
-    def export_bookmark(self, fp):
+    def export_bookmark(self, file_path, export_tags=None):
         '''Export bookmarks to a Firefox
         bookmarks formatted html file.
 
-        Params: Path to file to export to
+        Params: file_path: Path to file to export to
+        export_tags: tag to export, None for all
         '''
-
         import time
 
-        if os.path.exists(fp):
-            resp = input('%s exists. Overwrite? (y/n): ' % fp)
+        if os.path.exists(file_path):
+            resp = input('%s exists. Overwrite? (y/n): ' % file_path)
             if resp != 'y':
                 return
 
         try:
-            f = open(fp, mode='w', encoding='utf-8')
+            f = open(file_path, mode='w', encoding='utf-8')
         except Exception as e:
             logger.error(e)
             return
 
         count = 0
         timestamp = int(time.time())
-        self.cur.execute('SELECT * FROM bookmarks')
+        query = 'SELECT * FROM bookmarks'
+        if export_tags:
+            query += " WHERE tags LIKE '%' || ? || '%'"
+            self.cur.execute(query, (export_tags[0],))
```

Comment:
We need to take care of all tags passed, not just the first one.  Use `parse_tags()` and then process.